app/models/edition.rb

```diff
@@ -2,6 +2,7 @@
 
 # Respresents the current state of a piece of content that was once or is
 # expected to be published on GOV.UK.
+#
 # It is a mutable concept that is associated with a variety of immutable
```

Comment:
Slightly ambiguous about what's providing the past information - might be clearer to say 'succession' instead of 'variety', and 'that together represents successive snapshots of the content'.